test/pkg.jl

```diff
@@ -184,7 +184,7 @@ temp_pkg_dir() do
 
         Pkg.pin("Example", v"0.4.0")
         @test Pkg.update() == nothing
-        Pkg.installed()["Example"] == v"0.4.0"
+        @test Pkg.installed()["Example"] == v"0.4.0"
```

Comment:
is this alone sufficient to test what was broken, or should there be a few more?